tests/resource_management/libraries/script/script.py

```diff
@@ -8,4 +8,5 @@ def get_config():
         return {'configurations':
                 {'node.properties': {},
                  'jvm.config': {'jvm.config': ''},
-                 'config.properties': {}}}
+                 'config.properties': {}},
+                 'clusterHostInfo': {'presto_worker_hosts': [], 'presto_coordinator_hosts': []}}
```

Comment:
This PR got closed when I merge the branch onto which that was based. Sorry about that!

This constraint is enforced at the metainfo.xml level where you specify that the cardinality of your coordinator should be exactly one. I can add a check here as well if you want but I think it's overkill.